source/documentation/administration_guide/topics/Increasing_NFS_Storage.adoc

```diff
@@ -10,6 +10,6 @@ To increase the amount of NFS storage, you can either create a new storage domai
 . Click menu:Storage[Domains].
 . Click the NFS storage domain's name. This opens the details view.
 . Click the *Data Center* tab and click *Maintenance* to place the storage domain into maintenance mode. This unmounts the existing share and makes it possible to resize the storage domain.
-. On the NFS server, resize the storage. For {enterprise-linux} 6 systems, see link:{URL_customer-portal}{URL_docs}{URL_lang-locale}{URL_product_rhel}6/html-single/Storage_Administration_Guide/index.html#s2-disk-storage-parted-resize-part[{enterprise-linux} 6 Storage Administration Guide]. For {enterprise-linux} 7 systems, see link:{URL_rhel_docs_legacy}html-single/Storage_Administration_Guide/index.html#s2-disk-storage-parted-resize-part[{enterprise-linux} 7 Storage Administration Guide].
+. On the NFS server, resize the storage. For {enterprise-linux} 6 systems, see link:{URL_customer-portal}{URL_docs}{URL_lang-locale}{URL_product_rhel}6/html-single/Storage_Administration_Guide/index.html#s2-disk-storage-parted-resize-part[{enterprise-linux} 6 Storage Administration Guide]. For {enterprise-linux} 7 systems, see link:{URL_rhel_docs_legacy}html-single/Storage_Administration_Guide/index.html#s2-disk-storage-parted-resize-part[{enterprise-linux} 7 Storage Administration Guide]. For {enterprise-linux} 8 systems, see link:{URL_rhel_docs_latest}/html-single/managing_file_systems/index#assembly_resizing-a-partition_managing-file-systems[Resizing a partition].
```

Comment:
Eli, the RHEL 8 URL is resolving with a double "//" before "html-single": 

`https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8//html-single/managing_file_systems/index#assembly_resizing-a-partition_managing-file-systems`